.github/workflows/test_tutorial_workflow.yml

```diff
@@ -39,6 +39,13 @@ jobs:
       - name: install cylc-flow
         run: pip install .[all]
 
+      - name: lint tutorials
+        run: |
+          cylc get-resources tutorial
+          for f in $(find "${HOME}/cylc-src" -name flow*.cylc -exec dirname {} \;);
+            do cylc lint $f -r style
+          done
+
```

Comment:
Or, we could add `cylc lint` into the `.validate` script in each dir. Possibly more flexible at it allows each test to manage its own codes/ignores.

This is what I did on #5773.